app/controllers/BoxNumberController.scala

```diff
@@ -82,4 +91,14 @@ class BoxNumberController @Inject()(identify: IdentifierAction,
     }
   }
 
+  private def createRadioButton(form: Form[_], values: Seq[String])(implicit messages: Messages): Seq[RadioItem] = {
+    values.map(keyValue =>
```

Comment:
`keyValue` doesn't represent the items in the `Seq`, `boxNumber` would be more approppriate.